docs/getting_started/installing_development_tools.md

```diff
@@ -0,0 +1,94 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+---
+
+(getting_started_installing_development_tools)=
+
+# Installing development tools
+
+To create and run workflows in Flyte, you must install Python, flytekit, flytectl, and Docker. Installing conda or another Python virtual environment manager is optional, but strongly recommended.
```

Comment:
I think our steps for onboarding should be just `pip install flytekit` any more tools and it gets too much... flytectl and docker are needed when we get to running on the demo cluster, right? can these steps be postponed until then?